excel/beginner/charts/types-of-charts.md

```diff
@@ -0,0 +1,63 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# Types of charts
+
+---
+## Content
+
+There are a variety of charts you can make in Excel.
+Some of the more famous ones are the `Pie`, `Bar` and `Columns` charts.
+
+We will use the same data to create the 3 charts.
+
+Data used:
+![chart-data](https://img.enkipro.com/5fb6d6cec47fd3c56f26b1e8349d8a17.png)
+
+### The `Pie` Chart
+-------------------------------
+
+The pie chart is circular and sliced into sections like a pie. The whole Pie represents 100% and each section is a percentage of the whole pie.
+
+Example chart which shows the total number of users for the first 10 days of December for a certain app.
+
+![pie-chart](https://img.enkipro.com/9b9faa9cfb972490ce6ade3a59dc235c.png)
+
+
+### The `Bar` Chart
+
+The bar chart is used to represent data with either horizontal or vertical bars.
+
+![bar-chart](https://img.enkipro.com/b549d8d08b4ab7f10b3946dc7f7f0729.png)
+
+
+### The `Column` Chart
+
+The column charts are used to horizontally represent data with vertical bars.
+
+![column-chart](https://img.enkipro.com/81d21f8992f86192897d3e6a646eac31.png)
+
+---
+## Practice
+
+Which of these is not a famous Excel Chart?

```

Comment:
```suggestion
Which of these is not a well known Excel Chart?

```

Reply:
But also this question is kind of subjective. Any chart can be well known is some group of people. 

We should definitely change this question. 

Maybe something like "If you had this data and wanted to find this X you would use ??? chart"